### PR TITLE
Sync latest image stream and templates

### LINKED ISSUE #2801: Update Metrics templates for v1.4
There are new metric templates available for v1.4 required for https://bugzilla.redhat.com/show_bug.cgi?id=1392343

The origin template can be found here: https://github.com/openshift/origin-metrics

and the enterprise one is available here: https://github.com/openshift/origin-metrics/blob/enterprise/metrics.yaml

### PR DESCRIPTION
**Pull request by sdodson**
Fixes #2801

### CODE REVIEW

**Comment by sdodson** [comment]
@mwringe PTAL at metrics changes

**Comment by sdodson** [comment]
@ewolinetz can you check the logging deployer changes?

**Comment by sdodson** [comment]
aos-ci-test

**Comment by mwringe** [comment]
@sdodson the metric changes look right.

File: roles/openshift_hosted_templates/files/v1.4/enterprise/logging-deployer.yaml [hunk 1]
```diff
@@ -200,13 +220,13 @@ items:
...
-    description: 'Specify prefix for logging components; e.g. for "registry.access.redhat.com/openshift3/logging-deployer:3.4.0", set prefix "registry.access.redhat.com/openshift3/"'
+    description: 'Specify prefix for logging components; e.g. for "registry.access.redhat.com/openshift3/logging-deployer:3.3.0", set prefix "registry.access.redhat.com/openshift3/"'
     name: IMAGE_PREFIX
     value: "registry.access.redhat.com/openshift3/"
   -
-    description: 'Specify version for logging components; e.g. for "registry.access.redhat.com/openshift3/logging-deployer:3.4.0", set version "3.4.0"'
+    description: 'Specify version for logging components; e.g. for "registry.access.redhat.com/openshift3/logging-deployer:3.3.0", set version "3.3.0"'
     name: IMAGE_VERSION
-    value: "3.4.0"
+    value: "3.3.0"
```
Comment (ewolinetz) [inline roles/openshift_hosted_templates/files/v1.4/enterprise/logging-deployer.yaml]: Shouldn't this stay 3.4?

Comment (sdodson) [inline roles/openshift_hosted_templates/files/v1.4/enterprise/logging-deployer.yaml]: fixed now, thanks!

**Pull request merged**